worker/src/RTC/TransportCongestionControlServer.cpp

```diff
@@ -218,11 +229,19 @@ namespace RTC
 				case RTC::RTCP::FeedbackRtpTransportPacket::AddPacketResult::MAX_SIZE_EXCEEDED:
 				{
 					// This should not happen.
-					MS_WARN_DEV("transport-cc feedback packet is exceeded");
+					MS_WARN_TAG(rtcp, "transport-cc feedback packet is exceeded");
 
 					// Create a new feedback packet.
 					this->transportCcFeedbackPacket.reset(new RTC::RTCP::FeedbackRtpTransportPacket(
 					  this->transportCcFeedbackSenderSsrc, this->transportCcFeedbackMediaSsrc));
+
+					// Set current packet count.
+					// NOTE: Do not increment it since the previous ongoing feedback
+					// packet was not sent.
+					this->transportCcFeedbackPacket->SetFeedbackPacketCount(
```

Comment:
Do we need to set this value? Doesn't already have the same value already?

Reply:
How? There is a .reset() just above.

Reply:
That's right :+1: